src/test/java/org/opentripplanner/smoketest/util/GraphQLClient.java

```diff
@@ -0,0 +1,118 @@
+package org.opentripplanner.smoketest.util;
+
+import static org.opentripplanner.smoketest.SmokeTest.mapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+import org.opentripplanner.smoketest.SmokeTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GraphQLClient {
+
+  static final Logger LOG = LoggerFactory.getLogger(SmokeTest.class);
```

Comment:
Change `SmokeTest.class` to `GraphQLClient.class`

Reply:
Thanks - done!